plugins/strict-connection-checker/src/toolboxCategories.js

```diff
@@ -0,0 +1,849 @@
+          'inputs': {
+            'FROM': {
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 1,
+                },
+              },
+            },
+            'TO': {
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 100,
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'math_random_float',
+          'kind': 'block',
+        },
+        {
+          'type': 'math_atan2',
+          'kind': 'block',
+          'inputs': {
+            'X': {
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 1,
+                },
+              },
+            },
+            'Y': {
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 1,
+                },
+              },
+            },
+          },
+        },
+      ],
+    },
+    {
+      'kind': 'category',
+      'name': 'Text',
+      'categorystyle': 'text_category',
+      'contents': [
+        {
+          'type': 'text',
+          'kind': 'block',
+          'fields': {
+            'TEXT': '',
+          },
+        },
+        {
+          'type': 'text_multiline',
+          'kind': 'block',
+          'fields': {
+            'TEXT': '',
+          },
+        },
+        {
+          'type': 'text_join',
+          'kind': 'block',
+        },
+        {
+          'type': 'text_append',
+          'kind': 'block',
+          'fields': {
+            'name': 'item',
+          },
+          'inputs': {
+            'TEXT': {
+              'shadow': {
+                'type': 'text',
+                'fields': {
+                  'TEXT': '',
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'text_length',
+          'kind': 'block',
+          'inputs': {
+            'VALUE': {
+              'shadow': {
+                'type': 'text',
+                'fields': {
+                  'TEXT': 'abc',
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'text_isEmpty',
+          'kind': 'block',
+          'inputs': {
+            'VALUE': {
+              'shadow': {
+                'type': 'text',
+                'fields': {
+                  'TEXT': '',
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'text_indexOf',
+          'kind': 'block',
+          'fields': {
+            'END': 'FIRST',
+          },
+          'inputs': {
+            'VALUE': {
+              'block': {
+                'type': 'variables_get',
+                'fields': {
+                  'VAR': {
+                    'name': 'text',
+                  },
+                },
```

Comment:
It looks like the blocks that include variable blocks are broken as well :/ Could you remove these too?